commons/commons-util/src/main/kotlin/ebi/ac/uk/io/FileUtils.kt

```diff
@@ -146,21 +146,22 @@ internal object FileUtilsHelper {
     }
 
     fun createFolderHardLinks(
-        source: Path,
-        target: Path,
+        sourcePath: Path,
```

Comment:
any reason to change names here? I mean is not that I disagree with name change but trying to figured out the reasoning.

Reply:
It's just to make it clearer that the source is the path where the hard link will be created and not the referenced file itself